app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -41,6 +41,11 @@ open class RouteAction(
         object AutofillSearchDialog : DialogFragment(R.string.autofill)
     }
 
+    enum class EditItemMenu(@StringRes val titleId: Int) {
```

Comment:
I fear this may be abusing RouteActions, and could lead to more routing bugs.  I see there are route actions for other dialogs, but I suspect that may be a source of existing routing bugs (e.g., weirdness with system Back).

I suggest starting with ItemDetail, and going to a RouteAction-derived when the view/fragment actually has to change (if ever).